config/routes.rb

```diff
@@ -186,7 +186,7 @@
     root to: "admin/organisations#index", as: :authenticated_agent_root, defaults: { follow_unique: "1" }
   end
 
-  { contact: "contact", mds: "mds" }.each do |k, v|
+  { contact: "contact", mds: "mds", accessibility: "accessibility" }.each do |k, v|
```

Comment:
Hahaha j’avais raté ça, c’est un peu trop malin pour ce que ça fait non? D’habitude c’est moi qui fais des trucs superdynamiques, power of the runtime, tout ça. 🙈

Reply:
Héritage. C'est pas moi qui aie mis ça en place. Je trouve ça un peu overkill effectivement. À réfactorer plus tard :p